content/docs/start/data-management/data-versioning.md

```diff
@@ -0,0 +1,294 @@
+---
+title: 'Get Started: Data Versioning'
+description: 'Get started with data and model versioning in DVC. Learn how to
+use a regular Git workflow for datasets and ML models, without storing large
+files in Git.'
+---
+
+# Get Started: Data Versioning
+
+How cool would it be to make Git handle arbitrarily large files and directories
+with the same performance it has with small code files? Imagine cloning a
+repository and seeing data files and machine learning models in the workspace.
+Or switching to a different version of a 100Gb file in less than a second with a
+`git checkout`. Think "Git for data".
```

Comment:
... Everything else was moved to the Data Versioning page.

Reply:
One could argue that [Storing and sharing](https://dvc.org/doc/start/data-management#storing-and-sharing) and [Retrieving](https://dvc.org/doc/start/data-management#retrieving) should stay in DM though...